go/mysql/binlog_event_common.go

```diff
@@ -101,6 +108,11 @@ func (ev binlogEvent) Length() uint32 {
 	return binary.LittleEndian.Uint32(ev.Bytes()[9 : 9+4])
 }
 
+// nextPosition the nextPosition field from the header
```

Comment:
nit: `nextPosition` -> `NextPosition`

Reply:
actually, you probably meant `NextPosition returns the nextPosition ...`

Reply:
True!